plugins/integration/providers/voicebase/VoicebaseClientHelper.php

```diff
@@ -213,6 +218,27 @@ public function deleteRemoteFile($entryId)
 		$curlResult = $this->sendAPICall($params);
 	}
 
+	private function trimRedundantString($str, $substr)
```

Comment:
@shpiNi  this logic is not readable. Please add a doc comment explaining what's happening here.